electrumsv/gui/qt/network_dialog.py

```diff
@@ -1136,12 +1123,13 @@ def update_list(self, items: List[ServerListEntry]) -> None:
             self.setItem(row_index, 1, item_1)
 
             item_2 = SortableServerQTableWidgetItem()
-            if list_entry.server_type == NetworkServerType.ELECTRUMX and self._network is not None:
-                if list_entry.data_electrumx == self._network.main_server and \
-                        not self._network.auto_connect():
-                    item_2.setIcon(self._lock_icon)
-                    item_2.setToolTip(
-                        _("This server is locked into place as the permanent main server."))
+            # if list_entry.server_type == NetworkServerType.ELECTRUMX and
```

Comment:
All commented out code should be tagged with `TODO(1.4.0)`

Reply:
And why it is commented out so that it can be identified as either something to address or obsolete and removable at some point.